networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/subnets/StableSubnetSubscriberTest.java

```diff
@@ -60,89 +62,42 @@ void shouldNotNotifyAnyChangeWhenNumberOfValidatorsDecrease() {
     ArgumentCaptor<Set<SubnetSubscription>> subnetSubscriptions =
         ArgumentCaptor.forClass(Set.class);
 
-    stableSubnetSubscriber.onSlot(ZERO, 2);
-    verify(validatorApiChannel).subscribeToPersistentSubnets(subnetSubscriptions.capture());
+    stableSubnetSubscriber.onSlot(ZERO);
```

Comment:
we are not changing number of validators in these test since this PR, looks not relevant

Reply:
Done